ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    let layout: ContactCollectionViewLayout
+    @objc public weak var contactCollectionDelegate: UICollectionViewDelegate?
+
+    @objc public var contactList: [PersonaData] = [] {
+        didSet {
+            if (oldValue.count == 0 && contactList.count > 0) || (oldValue.count > 0 && contactList.count == 0) {
+                updateHeightConstraint()
+            }
+        }
+    }
+
+    @objc public init() {
+        layout = ContactCollectionViewLayout()
+        layout.scrollDirection = .horizontal
+        super.init(frame: .zero, collectionViewLayout: layout)
+        configureCollectionView()
+        updateHeightConstraint()
+        register(ContactCollectionViewCell.self, forCellWithReuseIdentifier: ContactCollectionViewCell.identifier)
+        NotificationCenter.default.addObserver(self, selector: #selector(setNewHeightConstraint), name: UIContentSizeCategory.didChangeNotification, object: nil)
```

Comment:
nitpick: In ObjC and swift, the "set" prefix in method names is reserved for property setters (although swift does have a different mechanism to declare setters using the "set" keyword, this rule seems to still apply). A better name would be "setupHeightConstraint".

Reply:
Thanks for informing me of this convention!